### ISSUE TITLE
Broken output: mismatched context and cluster names

### ISSUE THREAD

**Issue by jglick** [superbrothers/kubectl-view-kubeconfig-plugin#37]
Using 2.2.4 via Krew. I set my current context to a GKE cluster (working acc. to `kubectl version`) and ran this tool and it generated output, but the result was unusable (according to the same command, with a new `$KUBECONFIG`):

```
Error in configuration: context was not found for specified context: gke_prj_us-central1-a_clstr
```

I had to hand-edit the output to make it work:

```diff
9c9
<     cluster: gke_prj_us-central1-a_clstr
---
>     cluster: cluster
13c13
< current-context: gke_prj_us-central1-a_clstr
---
> current-context: context
```

Basically it overrode `clusters[0].name` to be `cluster` and `contexts[0].name` to be `context`, which is fine so far as it goes, but also left `contexts[0].context.cluster` and `current-context` at their original values, causing them to refer to nonexistent entries.

Note that `users[0].name`_and_ `contexts[0].context.user` were both left alone, so this part did not need to edited.

**Comment by superbrothers** [comment]
Thank you for your report. I've released v2.2.5 that is fixed this issue.

**Comment by jglick** [comment]
Seems to work.
